Filtered list UI tabs (#81)
AY2122S1-CS2103T-T10-4/tp: docs/UserGuide.md

Lists all currently supports commands, and provides a link to this user guide.<br>
Format: `help`

### Listing anime: `list`

Shows a list of all anime names the user has added.<br>

> An additional status parameter can be specified to show only anime with that status.<br>

Reviewer: I think this would be clearer as it suggests to the user that the list command can switch between tabs, rather than listing animes w different statuses on the same tab. Maybe we should also make clear what the tabs are for in the UG later on.
```suggestion
An additional status parameter can be specified to switch tabs, showing only anime with that status.<br>
```

Format: `list [s/STATUS]`

- `STATUS` refers to the updated status of the anime.

> - Available statuses: "towatch", "watching", "finished"

Reviewer 2: ```suggestion
- Available statuses: `towatch`, `watching`, `finished`
```

Reviewer 2: im changing them to use code block in my end, so should change here to make it consistent

> - Shortforms: "t", "w", "f"

Reviewer 2: ```suggestion
- Shortforms: `t`, `w`, `f`
```
